lib/shopify/customer.ex

```diff
@@ -112,6 +174,28 @@ defmodule Shopify.Customer do
 
   @doc """
   Update a customer.
+
+  ## Examples
+      iex> session = Shopify.new_private_session("my-store", "xxxxx", "yyyyy")
```

Comment:
No need to add session init in examples. It's beyond the scope of what the example should cover.